src/js/models/recipientLanding/BaseRecipientLandingRow.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * BaseRecipientLandingRow.js
+ * Created by David Trinh on 7/3/18
+ */
+
+import { formatMoney } from 'helpers/moneyFormatter';
+
+/* eslint-disable object-shorthand */
+const BaseRecipientLandingRow = {
+    parse: function (data) {
```

Comment:
Could you rename this function `populate` to be consistent with other v2 models?